dask/dataframe/io/parquet/core.py

```diff
@@ -28,6 +28,7 @@ def read_parquet(
     storage_options=None,
     engine="auto",
     gather_statistics=True,
+    infer_divisions=False,
```

Comment:
I think that my original intent was to remove this keyword, and replace it with `gather_statistics=`

Reply:
Oops - I had meant to remove this.